ingestion/functions/parsing/brazil_espirito_santo/espirito_santo.py

```diff
@@ -173,9 +168,11 @@ def convert_preexisting_conditions(lung: str, kidney: str, diabetes: str,
         comorbidities.append(_COMORBIDITIES_MAP["ComorbidadeObesidade"])
 
     if comorbidities:
+        preexistingConditions["hasPreexistingConditions"] = True
         preexistingConditions["values"] = comorbidities
-
-    return preexistingConditions
+        return preexistingConditions
+    else:
+        return None
 
```

Comment:
Just noting that most of these changes aren't about bad age entries :). It helps to keep the PR description, commit messages, and changes in sync so that the project history is an accurate record of the changes made, and so that we reviewers know what we're looking for in the changes.

Reply:
Ok thanks, will make sure I have separate commits in the future.